torch/distributed/distributed_c10d.py

```diff
@@ -197,32 +197,80 @@ def __getattribute__(self, key):
 
 reduce_op = _reduce_op()
 
+# Not including _default_pg_init_method cuz it's dead code (nothing uses it in PT)
+class World:
```

Comment:
add a docstring if this is meant to be public and user-facing?

Reply:
This should be a private API for now. I'll change the visibility.